lib/recaptcha/token.rb

```diff
@@ -0,0 +1,19 @@
+module Recaptcha
+  module Token
+    require 'json'
+
+    def self.secure_token
```

Comment:
in addition to https://github.com/ambethia/recaptcha/commit/1a151e871df7f3a6c96994549ae07f63c52a3735#diff-35bfa94484a999878842a1178b1216b3R70 ?